### ISSUE TITLE
LifeForm Inventory

### ISSUE THREAD

**Issue by CJPoll** [CJPoll/exorcist#12]
Operations I think we should be able to support:
1. Add item
2. Remove item
3. Stackable items?

Not necessarily as a class - I think Rybec's idea of a list or set would make a lot of sense. We need functions to be able to support these operations (and any others we think of)

**Comment by JakobHartman** [comment]
If we use a list we can just have multiple of an item but in a players perspective they stack

**Comment by CJPoll** [comment]
Yeah, I just did it as a set because our operations seemed to be more in line with that. We can implement stacking as either a list with duplicate items automatically being stacked, or we can just update a stackable item's count field. I'm fine with either.

For reference, here's a link to the the pull request where I implemented inventory: https://github.com/CJPoll/exorcist/pull/14

Switching it to a list would be trivial, and if you look at the PR's commit history, you'll see that's how I initially did it. If you want to change it back to a list and implement stacking, I'd love to see it 😄
